guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -7,5 +7,11 @@
 [id="foreman-upgrade-warnings"]
 == Upgrade Warnings
 
+=== Fedora/Red Hat Atomic support dropped
```

Comment:
```suggestion
=== Fedora/Red Hat Atomic is no longer supported
```
Can this be better?

Reply:
Is that really better? I always think headers should be short.

Looking at writers here.

Reply:
When we drop things, we mostly use `no longer supported`.